db/migrate/20150429192808_add_scope_to_sequences.rb

```diff
@@ -0,0 +1,5 @@
+class AddScopeToSequences < ActiveRecord::Migration
+  def change
+    add_column :sequences, :scope, :string
```

Comment:
There's not going to be more than two rows. I don't think a full tablescan will be that painful.